pages/glossary.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <h1 class="text-center text-2xl md:text-3xl lg:text-4xl mb-6">Glossary</h1>
+    <p class="text-center mb-3">
+      This glossary contains unusual or specialist words used across this website, including in filters and explanations, to help you better understand the terms and their meanings.
```

Comment:
The _including in filters and explanations_ bit seems a redundant. I think we should take this bit out as it is covered by the preceding part of the sentence.